website/src/components/Features/styles.module.css

```diff
@@ -35,10 +35,17 @@
 }
 
 .featureContent ul {
-  width: 70%;
+  list-style-type: '✔';
```

Comment:
BTW, I just realized this uses 2 spaces for indentation, whereas in the file of the previous commit 4 spaces are used. We should probably align on either indentation for the same file type.

Reply:
Yes, I would like to introduce prettier for the CSS and JS files.